content/200-orm/100-prisma-schema/20-data-model/10-models.mdx

```diff
@@ -425,7 +425,11 @@ Refer to the [relations documentation](relations) for more examples and informat
 
 ### Native types mapping
 
-Version [2.17.0](https://github.com/prisma/prisma/releases/tag/2.17.0) and later support **native database type attributes** (type attributes) that describe the underlying database type:
+<versioninfo>
+  Version [2.17.0](https://github.com/prisma/prisma/releases/tag/2.17.0) and
+  later support **native database type attributes** (type attributes) that
+  describe the underlying database type:
+</versioninfo>
```

Comment:
This is a good example where I think the docs would benefit from rewriting this in a way without mentioning the version information, and than add that information in parens after the sentence: "(This is available since version 2.17.0)".